Neos.Neos/Classes/Fusion/ConvertUrisImplementation.php

```diff
@@ -114,16 +115,18 @@ public function evaluate()
             ), 1382624087);
         }
 
+        /** @var RenderingMode $renderingMode */
+        $renderingMode = $this->runtime->fusionGlobals->get('renderingMode');
```

Comment:
I probably mix things up, but didn't we add a method to the Fusion runtime to retrieve those things in a type-safe manner?
If not, I would suggest to replace the implicit type annotation with an explicit check like
```suggestion
        $renderingMode = $this->runtime->fusionGlobals->get('renderingMode');
        assert($renderingMode instanceof RenderingMode);
```
or
```suggestion
        $renderingMode = $this->runtime->fusionGlobals->get('renderingMode');
        if (!$renderingMode instanceof RenderingMode) {
            throw new \RuntimeException('Failed to extract rendering mode from Fusion context', 1700216167);
        }
```

Reply:
i hate to know youre right

Reply:
Not a big thing, but if you agree and apply the suggestion, I'd merge this